### ISSUE TITLE
Resize Example - Height only gets recalculated when the scrollbar appears

### ISSUE THREAD

**Issue by nozpheratu** [oyvindkinsey/easyXDM#210]
I was following the easyXDM resize example shown here: http://consumer.easyxdm.net/current/example/resize_iframe.html

I have the same iframe and JavaScript setup, however it didn't quite work for me. In side of the content iframe (the one retrieved by the provider iframe) we have the following line:

``` javascript
window.onload = function(){                      
    parent.socket.postMessage(document.body.clientHeight || document.body.offsetHeight ||   document.body.scrollHeight);            
};        
```

It works fine when a new page is loaded within the inner iframe with content larger in height than the previous page (in other words, whenever there is a scroll bar). However when you navigate to a page with less content and a smaller height(no scrollbar appears) a new height is never calculated for the new page. 

In other words a new height is only ever calculated when a scrollbar is present. Is there a fix for this so page sizes can be adjusted for both smaller and larger pages than the previous?

A temporary/hacky fix for this is to force the scrollbar to appear by calling `parent.socket.postMessage` twice within inner iframe. The first call sets it to a very small height of say, 10 pixels which forces the scrollbar to appear. The second call sends the document height. I don't like this solution as it can produce fickle results, i.e. doesn't always set the size quite right. 

Edit: Scratch that last bit, I thought that hotfix worked but it only actually seems to work when I'm testing locally and not in my production environment where the iframe is loaded on a separate domain.
